tests/functional/testCaptivePortal.js

```diff
@@ -58,8 +58,13 @@ describe('Captive portal', function() {
     assert.equal(vpn.lastNotification().title, null);
   });
 
-  it('Captive portal in the Post authentication view', async () => {
-    await vpn.authenticateInApp();
+  it('Captive portal in the Post authentication view', async function () {
+    //Post auth view does not exist in new onboarding
+    if (await vpn.isFeatureEnabled("newOnboarding")) {
+      this.skip();
```

Comment:
I don't get it. Shouldn't we remove these tests?

Reply:
I felt a bit reluctant to remove anything related to old onboarding (post auth and telemetry screens) until the new onboarding is released and is confirmed working well enough that we don't have to revert it.

I plan on removing everything related to the old onboarding, including tests, when we delete the feature flag

Reply:
The real question to me was should we be running tests on the old onboarding, even when the new onboarding is disabled. I concluded that it just doesn't make sense to run functional tests on code that isn't active (without going into dev settings), it would just be too confusing with all of the permutations, so thats the reason I am skipping most post auth/telemetry screen tests until they can be removed

Reply:
If the code is still in there we should keep on testing it even though it is not enabled. In this case I suggest flipping the feature flag on for all tests. Think about it this way: we don't it to break in case we need to re-enable it. When the whole thing is removed we remove the tests as well.